src/FeatureSwitcher.VstsConfiguration/UrlWrapper.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace FeatureSwitcher.VstsConfiguration
 {
-    internal class UrlWrapper
+    public class UrlWrapper
```

Comment:
Is there a reason you made this public?

Reply:
I just updated the package references. 
Rest of the code is more or less from your is more or less from your features/net-standard-support-branch.